src/main/resources/view/ClientInfoPanel.fxml

```diff
@@ -1,32 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
However, using the javafxml to set the styling (colour, border colour, border radius, background radius, text colour for the parts of the fxml) is not typical as it makes it hard to change. Do you think its possible to refactor these into the CSS?